frontend/cypress/integration/navbar.spec.js

```diff
@@ -0,0 +1,79 @@
+import faker from 'faker'
+
+describe('Validates `Add Project` behavior', () => {
+  const username = faker.name.firstName()
+  const email = faker.internet.email()
+  const password = '123456'
+
+  before(() => {
+    cy.visit('/')
+    cy.createUser(username, email, password)
+  })
+
+  beforeEach(() => {
+    cy.signOut()
+  })
+
+  it('unauthenticated user', () => {
```

Comment:
Would be good to write all these test names as grammatically correct sentences beginning with "it: "it does something..."